java/src/org/openqa/selenium/grid/distributor/GridModel.java

```diff
@@ -47,6 +47,8 @@
 import org.openqa.selenium.grid.data.SessionClosedEvent;
 import org.openqa.selenium.grid.data.Slot;
 import org.openqa.selenium.grid.data.SlotId;
+import org.openqa.selenium.grid.data.SlotMatcher;
```

Comment:
No changes to the `GridModel` are needed anymore.

Reply:
updated